persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/member/MemberDAO.kt

```diff
@@ -147,12 +138,17 @@ internal class MemberDAOImpl internal constructor(
     ) = withContext(coroutineContext) {
         memberQueries.transaction {
             userQueries.updateUserConnectionStatus(status, member.user)
-            val recordDidNotExist = userQueries.selectChanges().executeAsOne() == 0L
-            if (recordDidNotExist) {
+            val userRecordDidNotExist = userQueries.selectChanges().executeAsOne() == 0L
+            if (userRecordDidNotExist) {
                 userQueries.insertOrIgnoreUserIdWithConnectionStatus(member.user, status)
             }
             conversationsQueries.updateConversationType(ConversationEntity.Type.ONE_ON_ONE, conversationID)
-            memberQueries.insertMember(member.user, conversationID, member.role)
+            val conversationRecordExist = conversationsQueries.selectChanges().executeAsOne() != 0L
+            if(conversationRecordExist) {
+                memberQueries.insertMember(member.user, conversationID, member.role)
+            } else {
+                kaliumLogger.e("conversation $conversationID doest not exist for user ${member.user}")
```

Comment:
```suggestion
                kaliumLogger.w("conversation $conversationID doest not exist for user ${member.user}")
```

Maybe we can just log it as a warning, given I see we are not raising an error on this 🤔